### ISSUE TITLE
Site Proposal

### ISSUE THREAD

**Issue by VirtDev337** [slashrelativepath/relativepath.tech#1]
Outlined below are a few recommendations for the proposed website.  As this was discussed to a degree in the discord server, some of the points reflect recommendation contributions from several individuals.

## Basic Description
The site would be a landing page that outlines the purpose, or mission, of what is being attempted.  Maybe include a history and explanation that the focus isn't tooling, but the concepts behind the tools.  

### Inclusion
- scheduling 
- an approximation of a syllabus outlining the theories/practices covered.  

- Finally, links to: 
  - Discord discussion
  - Audio/video channel 
  - Youtube channel 
  - Contact information that can email a group of individuals, so no one person is swamped with information and application requests.  
  - Social Media that exists for the group
  - Donation (paypal or other such financial institution)

### Hosting
- To start, we could use github pages
- Voted sites can be deployed to pythonanywhere or other free hosting site
- Regardless of cohort efforts, a site should be registered with relativepath.tech and published as the introduction to the idea and course

**Comment by stephenlauck** [comment]
I would say we don't want to post a link to our discord nor the schedule for now. We want to keep it just informational and have any "call to action" be just to email stephen@relativepath.tech and/or jason@relativepath.tech.

**Comment by kensonjohnson** [comment]
> I would say we don't want to post a link to our discord nor the schedule for now. We want to keep it just informational and have any "call to action" be just to email [stephen@relativepath.tech](mailto:stephen@relativepath.tech) and/or [jason@relativepath.tech](mailto:jason@relativepath.tech).

Does this mean we have an acceptance pipeline?

**Comment by kensonjohnson** [comment]
Does this mean that relativepath.tech will not be static? As in, the website will be changed as a new one is generated every 8 weeks?

**Comment by stephenlauck** [comment]
> Does this mean that relativepath.tech will not be static? As in, the website will be changed as a new one is generated every 8 weeks?

We should start with one site, then we can replace with new sites as part of the  program.

**Comment by stephenlauck** [comment]
> > I would say we don't want to post a link to our discord nor the schedule for now. We want to keep it just informational and have any "call to action" be just to email [stephen@relativepath.tech](mailto:stephen@relativepath.tech) and/or [jason@relativepath.tech](mailto:jason@relativepath.tech).
> 
> Does this mean we have an acceptance pipeline?

yeah, just laying out a bit of acceptance. we should fully pipeline the deployment.

**Comment by kensonjohnson** [comment]
What type of site will be produced at the end of a cycle? 
- Static site using Vanilla HTML / JavaScript / CSS
- Static site using frontend framework (e.g. React)
- Web server (ExpressJS, Django, etc) serving static HTML files
- Web server with view engine
- Full "batteries included" framwork (like NextJS)

**Comment by VirtDev337** [comment]
I'd say html, vanilla js, maybe a css framework... since it's going to be scrapped every few weeks.

Still think a main site that has a professional appearance and is always available should be the landing page.   

It can be built out with more pages,  possibly including scripts for the current cohort,  since there are many ways to accomplish the same thing.

**Comment by kensonjohnson** [comment]
I think a solid foundation for the site would be using React and some opinionated styling framework. React compiles to a static target and is easily hosted on GH Pages for free. When we move to add features at a later date, having the code already in a React project will make the transition easier.
